### ISSUE TITLE
os: Add example showing how to read data from stdin

### ISSUE THREAD

**Issue by kevinburke** [golang/go#22421]
A pretty common task, especially for new programmers, is to ask the user to type in some value and then act on it. (think "interactive maze game", "determine if word is palindrome", etc). It would be nice if we had an example in the `os` package showing how to do this.

**Comment by dpinela** [comment]
I don't think that belongs in the `os` package documentation (a new programmer is unlikely to look there first for help). What you suggest would belong more in a tutorial, like the [Go tour](https://tour.golang.org), which already has a lot of basic examples, though nothing about reading console input.

**Comment by kevinburke** [comment]
Ideally we'd add it in both places.

**Comment by rucuriousyet** [comment]
If this hasn't been taken yet, I should be able to do this. The line example (provided here: https://golang.org/pkg/bufio/#Scanner) is perfect for what I think you are looking for. I may adapt the example but it should be a pretty quick addition to the docs...

**Comment by crvv** [comment]
I don't think a new programmer should use `os` or `bufio` to do this.
`fmt` is much simpler.
```
package main

import "fmt"

func main() {
    var x, y int
    fmt.Scan(&x, &y)
    fmt.Println(x + y)
}
```
